test_helpers/pool_helper.go

```diff
@@ -215,16 +256,23 @@ func SetClusterRO(dialers []tarantool.Dialer, connOpts tarantool.Opts,
 		return fmt.Errorf("number of servers should be equal to number of roles")
 	}
 
+	ctx, cancel := GetConnectContext()
+	defer cancel()
```

Comment:
It looks better to add a context param to the `SetClusterRO` call instead of the hidden-confusing-context-timeout.

We could do that since we don't guarantee a backward compatibility for the `test_helper` package.


```
func SetClusterRO(dialers []tarantool.Dialer, connOpts tarantool.Opts,
	roles []bool) error {
```
->
```
func SetClusterRO(ctx context.Context, dialers []tarantool.Dialer,
	connOpts tarantool.Opts, roles []bool) error {
```

Reply:
I understand that we can modify API here, but it doesn't seem to be the case.
Context is not a kind of global entity, it is always created in some function and then passed into some function that accepts context. Does this make call of such a context-creator-function as hidden-confusing-context? For me context here is just a kind of implementation details. It doesn't seem that caller need to know about it and I see no problem that he doesn't.

Reply:
This is a matter of expected behavior from a call from a code-reader point of view. I insist on this change and the addition of a cotext as a first call argument. Please, add it.

Reply:
Done.